### PR TITLE
Send more Yii2 standard context to Sentry. Make Target a bit more extendable.

### PR DESCRIPTION
**Pull request by sp-niemand**

### CODE REVIEW

File: src/Target.php [hunk 1]
```diff
@@ -63,10 +74,30 @@ public static function filterMessages($messages, $levels = 0, $categories = [],
...
+        return [];
+    }
+
+    /**
+     * @return array
+     * @see https://docs.getsentry.com/on-premise/learn/context/
+     */
+    protected function getExtraData()
+    {
+        return array_intersect_key($GLOBALS, array_flip($this->logVars));
```
Comment (m8rge) [inline src/Target.php:91]: Does this conflict with https://github.com/E96/yii2-sentry/blob/master/src/SentryHelper.php#L13 ?

Comment (sp-niemand) [inline src/Target.php:91]: No. These helper methods add data on a Raven client context level.
Methods I added inject data to the array which gets sent by the client. It's on another level.

Comment (m8rge) [inline src/Target.php:91]: Sentry capture global vars automatically:
2015-10-07 18-01-18 undefined property stdclass row

Why do you need to capture them twice?

File: src/Target.php [hunk 2]
```diff
@@ -63,10 +74,30 @@ public static function filterMessages($messages, $levels = 0, $categories = [],
     }
 
     /**
-     * Exports log [[messages]] to a specific destination.
+     * @return array
+     * @see https://docs.getsentry.com/on-premise/learn/context/
+     */
+    protected function getTagsData()
+    {
+        return [];
```
Comment (m8rge) [inline src/Target.php:82]: Do you extend this class and override this method to add tags? How do you use this method?

File: src/Target.php (hunk 1 above)
Comment (sp-niemand) [inline src/Target.php:91]: You are right. Looks like "logVars" should be empty by default.
But I think we need $_SERVER logged for CLI scripts.

File: src/Target.php (hunk 2 above)
Comment (sp-niemand) [inline src/Target.php:82]: Yeah, that was the idea. But firstly I screwed up merging these custom tags with the "category" in export(), and secondly this method is not useful globally for every log record exported by the target, only for each log record separately.

**Pull request closed without merging**